app/scripts/modules/netflix/chaosMonkey/chaosMonkeyExceptions.directive.js

```diff
@@ -36,7 +36,9 @@ module.exports = angular
           this.accounts = details;
           this.regionsByAccount = {};
           details.forEach((account) => {
-            this.regionsByAccount[account.name] = ['*'].concat(account.regions.map((region) => region.name));
+            if (account.regions) {
+              this.regionsByAccount[account.name] = ['*'].concat(account.regions.map((region) => region.name));
+            }
```

Comment:
I think this is the real issue - the Titus accounts do not have `regions`, so that `map` throws an exception when it hits them. Not sure it's a matter of async loading.